test-no-uncommitted-doc-changes.in

```diff
@@ -20,7 +20,7 @@ if [ "$PLONE_VERSION" = "5.0.x" ]; then
     exit 0
 fi
 
-changes=$(git diff --exit-code $DUMPS_DIR)
+changes=$(git diff --ignore-space-at-eol --exit-code $DUMPS_DIR)
```

Comment:
My editor removes whitespace at EOL as well. Though, I usually do not touch the auto-generated files at all. :)